src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriterSettings.cs

```diff
@@ -5,9 +5,8 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
-using System.Runtime.Versioning;
 using System.Diagnostics.CodeAnalysis;
-
+using System.Runtime.CompilerServices;
```

Comment:
btw if you're planning on more PRs consider investigating usage of the defines like `HIDE_XSL` below - I suspect they're always set or unset and not used in multiple projects so we could potentially remove all the `#if`s